### PR TITLE
Adds INP-related case studies

### PR DESCRIPTION
**Pull request by malchata**
Adds INP case studies for:
- redBus
- Economic Times

### CODE REVIEW

File: src/site/content/en/blog/economic-times-inp/index.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+The traffic received on topic pages represents a significantly smaller portion of overall traffic. Hence, it was an ideal place for experimentation. The CrUX results along with the business outcomes were very encouraging, and led us to expand our efforts across the entire website to reap further benefits.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/pkPEYIQIeEDKdIKOCFSY.png", alt="A screenshot of INP distributions as visualized in CrUX over a period of four months, starting in July 2022 and ending in October 2022. Values within the 'poor' and 'needs improvement' thresholds declined somewhat, while the values within the 'good' threshold increased.", width="800", height="190" %}
+</figure>
+
+### Akamai mPulse TBT Analysis
+
+We use [Akamai mPulse](https://www.akamai.com/products/mpulse-real-user-monitoring) as our RUM solution. Though TBT is a lab metric, we observed a consistent decrease in TBT, clearly mapping to the outcomes of our efforts to reduce INP. As can be seen in the screenshot below, TBT values eventually dropped from approximately 5 seconds in to around 200 milliseconds in the field.
```
Comment (tunetheweb) [inline src/site/content/en/blog/economic-times-inp/index.md:147]: Confused about this sentence. If mPulse is RUM, and TBT is lab, how are you measuring TBT in mPulse?

Not big enough to hold up publishing but one to follow up afterwards.

File: src/site/content/en/blog/economic-times-inp/index.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: Economic Times quest for fixing INP
```
Comment (tunetheweb) [inline src/site/content/en/blog/economic-times-inp/index.md:3]: Shame no image for this post.

File: src/site/content/en/blog/redbus-inp/index.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    A screenshot of the logging system used by redBus to analyze INP values collected from the field. (<a href="https://web-dev.imgix.net/image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/s4JxEJgRhQAYZRSi7X3m.png" rel="noopener" target="_blank">Click for a higher resolution version of this image</a>.)
+  </figcaption>
+</figure>
+
+## Use cases
+
+When users search for fares on the redBus website, they can change the date on the search page to filter selected fares for the desired destination. The interaction to change the date on this page was slow, and was a cause of poor INP.
+
+**TODO: INSERT GIF**
+
```
Comment (tunetheweb) [inline src/site/content/en/blog/redbus-inp/index.md:66]: TODO? There's a Gif below, so should that be here? Or is that another GIF?

File: src/site/content/en/blog/redbus-inp/index.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+3. Ensure that responses from their API were as light as possible to ensure fast data transfer to the front end.
+
+## The journey
+
+redBus categorized interaction latency in two ways:
+
+1. Interaction latency caused by blocking JavaScript on the client. When interactions use excessive JavaScript that blocks the main thread, rendering is delayed, and this affects a page's INP.
+2. Network latency caused by API calls. While network activity is not something that INP measures, an interaction reliant on a call to a remote API can feel sluggish on slower networks, or if requests result in large responses.
+
+redBus was initially quite confident that the INP for their website would be good, but the [Real User Monitoring (RUM)](https://en.wikipedia.org/wiki/Real_user_monitoring) data for this metric at the 95th percentile told a different story.
```
Comment (tunetheweb) [inline src/site/content/en/blog/redbus-inp/index.md:44]: They use web-vitals.js as their RUM but say they are looking at 95th percentile. Web-vitals.js uses 75th percentile so is this a typo?

**Review by tunetheweb — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/economic-times-inp/index.md (hunk 1 above)
Comment (tunetheweb) [reply src/site/content/en/blog/economic-times-inp/index.md:147]: Confirmed with some Akamai folks I'm with that it DOES measure TBT in the field.

Suggest slight tweak to the wording:

```suggestion
We use [Akamai mPulse](https://www.akamai.com/products/mpulse-real-user-monitoring) as our RUM solution, which measures TBT in the field. We observed a consistent decrease in TBT, clearly mapping to the outcomes of our efforts to reduce INP. As can be seen in the screenshot below, TBT values eventually dropped from approximately 5 seconds in to around 200 milliseconds in the field.
```

File: src/site/content/en/blog/redbus-inp/index.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: How redBus improved their website's Interaction to Next Paint (INP) and increased sales by 7%
+subhead: |
+  INP optimizations helped redBus increase sales by ~7%
```
Comment (rachelandrew) [inline src/site/content/en/blog/redbus-inp/index.md:5]: style guide doesn't like ~, can we use a word here?

File: src/site/content/en/blog/redbus-inp/index.md (hunk 4 above)
Comment (tunetheweb) [reply src/site/content/en/blog/redbus-inp/index.md:44]: Ignore. Percentile is calced after collection. So they could use 95th percentile.

File: src/site/content/en/blog/redbus-inp/index.md (hunk 5 above)
Comment (malchata) [reply src/site/content/en/blog/redbus-inp/index.md:5]: Yeah, I'll replace it with "about" and push a new change right away.

File: src/site/content/en/blog/economic-times-inp/index.md (hunk 2 above)
Comment (malchata) [reply src/site/content/en/blog/economic-times-inp/index.md:3]: Yeah, but the prior Economic Times one we did last year didn't have a hero image, so I decided to just leave it for now.

**Review by rachelandrew — APPROVED** [review]

**Review by tunetheweb — APPROVED** [review]

**Pull request merged**